src/data/nat/totient.lean

```diff
@@ -7,6 +7,7 @@ import algebra.big_operators.basic
 import data.nat.prime
 import data.zmod.basic
 import ring_theory.multiplicity
+import data.nat.factorization
```

Comment:
This import doesn't appear to be necessary, at least on my version of mathlib.

Reply:
Ah, I hadn't merged master since #12033 went in, which adds this import to `ring_theory/multiplicity`.